big-dipper/imports/api/analytics/server/methods.js

```diff
@@ -135,346 +137,346 @@ if (Meteor.isServer) {
       const sales = Analytics.find(
         {
-          type: 'Sale'
+          type: "Sale",
         },
         {
           sort: { time: -1 },
           limit: limitVal,
-          skip: offsetVal
+          skip: offsetVal,
         }
-      ).fetch()
+      ).fetch();
 
       for (let i = 0; i < sales.length; i++) {
-        const buyerUsername = getUserNameInfo(sales[i]?.to)
-        const sellerUsername = getUserNameInfo(sales[i].from)
+        const buyerUsername = getUserNameInfo(sales[i]?.to);
+        const sellerUsername = getUserNameInfo(sales[i].from);
 
-        sales[i].to = buyerUsername?.username?.value
-        sales[i].from = sellerUsername?.username?.value
+        sales[i].to = buyerUsername?.username?.value;
+        sales[i].from = sellerUsername?.username?.value;
       }
-      return sales
+      return sales;
     },
-    'Analytics.getSaleOfAllTime': async function () {
+    "Analytics.getSaleOfAllTime": async function () {
       // sale of all time
       const sale = Analytics.find(
         {
-          type: 'Sale',
-          coin: SalesAnalyticsDenom
+          type: "Sale",
+          coin: SalesAnalyticsDenom,
         },
         {
           sort: { amount: -1, time: -1 },
-          limit: 1
+          limit: 1,
         }
-      ).fetch()
+      ).fetch();
 
-      return extractSaleFromSales(sale)
+      return extractSaleFromSales(sale);
     },
-    'Analytics.getSaleOfTheDay': async function () {
-      const start = new Date()
-      start.setDate(start.getDate() - 1)
-      start.setHours(0, 0, 0, 0)
-      const startDate = getFormattedDate(start)
+    "Analytics.getSaleOfTheDay": async function () {
+      const start = new Date();
+      start.setDate(start.getDate() - 1);
+      start.setHours(0, 0, 0, 0);
+      const startDate = getFormattedDate(start);
 
-      const end = new Date()
-      end.setDate(end.getDate() + 1)
-      end.setHours(0, 0, 0, 0)
-      const endDate = getFormattedDate(end)
+      const end = new Date();
+      end.setDate(end.getDate() + 1);
+      end.setHours(0, 0, 0, 0);
+      const endDate = getFormattedDate(end);
 
       // sale of today
       const sale = Analytics.find(
         {
-          type: 'Sale',
+          type: "Sale",
           coin: SalesAnalyticsDenom,
-          time: { $gte: startDate, $lt: endDate }
+          time: { $gte: startDate, $lt: endDate },
         },
         {
           sort: { amount: -1 },
-          limit: 1
+          limit: 1,
         }
-      ).fetch()
+      ).fetch();
 
-      return extractSaleFromSales(sale)
+      return extractSaleFromSales(sale);
     },
-    'Analytics.getSalesGraph': async function () {
-      const start = new Date()
-      const end = new Date()
-      start.setDate(start.getDate() - 7)
-      end.setDate(end.getDate() - 6)
+    "Analytics.getSalesGraph": async function () {
+      const start = new Date();
+      const end = new Date();
+      start.setDate(start.getDate() - 7);
```

Comment:
"7" should be in a constant.

Reply:
Resolved.
![image](https://user-images.githubusercontent.com/101688102/183139989-b5286d8d-29b0-471a-a3e7-e0f03de6de42.png)

Updated to this.